admin/nodes/views.py

```diff
@@ -1,17 +1,23 @@
 from website.project.model import Node
 
-from admin.base.views import GuidFormView
+from admin.base.views import GuidFormView, GuidView
 from admin.nodes.templatetags.node_extras import reverse_node
 from .serializers import serialize_node
 
 
 class NodeFormView(GuidFormView):
-    template_name = 'nodes/node.html'
+    template_name = 'nodes/search.html'
     object_type = 'node'
 
-    def get_guid_object(self):
-        return serialize_node(Node.load(self.guid))
-
     @property
     def success_url(self):
         return reverse_node(self.guid)
+
+
+class NodeView(GuidView):
+    template_name = 'nodes/node.html'
+    context_object_name = 'node'
+
+    def get_object(self, queryset=None):
+        self.guid = self.kwargs.get('guid', None)
```

Comment:
As suggested in https://github.com/CenterForOpenScience/osf.io/pull/5040/files#r55427109, I don't think this is necessary.